### DISCUSSION TITLE
Broken Links reporting

### DISCUSSION THREAD

**Discussion by efcjimbo** [elrumo/macOS_Big_Sur_icons_replacements#1513] [General]
No way to report broken links or links downloading the wrong file. 
There is a sea of bad links on this site with no way to report them.

**Comment by crus1313** [comment]
All Pearcleaner by alessandrx

https://parsefiles.back4app.com/JPaQcFfEEQ1ePBxbf6wvzkPMEqKYHhPYv8boI1Rc/07c3824cb53b13174e393d8cb172557c_Pearcleaner__for_big_sur__without_logo_.icns

**Comment by scottkuhl** [comment]
[Applications Folder by epicdude2121](https://macosicons.com/#/?icon=T3fUG0sfgZ)

https://parsefiles.back4app.com/JPaQcFfEEQ1ePBxbf6wvzkPMEqKYHhPYv8boI1Rc/03ddfe4e564207329eb7fd62ce07cfa9_Applications_Folder.icns

**Comment by AClownfa** [comment]
[USB icon uploaded by pannus_taciturnus](https://macosicons.com/#/?icon=XK5qZwLPLg)

https://parsefiles.back4app.com/JPaQcFfEEQ1ePBxbf6wvzkPMEqKYHhPYv8boI1Rc/37984bd89665d537574e748d7295121f_USB.icns

**Comment by MitchTalmadge** [comment]
Terminal icon
https://macosicons.com/#/?icon=2wpyg2FfPi

**Comment by einsteinhead** [comment]
https://parsefiles.back4app.com/JPaQcFfEEQ1ePBxbf6wvzkPMEqKYHhPYv8boI1Rc/12589bf90569f6570b861a5018e8b128_Audiofuse.icns

**Comment by leopoldchou** [comment]
https://parsefiles.back4app.com/JPaQcFfEEQ1ePBxbf6wvzkPMEqKYHhPYv8boI1Rc/0f6f2f22d1d1198052648f4461ff5c8e_YlK9JVqHu6.icns

**Comment by Maicolev** [comment]
how can I help fix these links?

https://parsefiles.back4app.com/JPaQcFfEEQ1ePBxbf6wvzkPMEqKYHhPYv8boI1Rc/1857bb1ce1c72e14a9e8a266e8686168_DataGrip__with_Background_.icns

**Comment by AstonFox** [comment]
Hi
Can someone fix this link: https://parsefiles.back4app.com/JPaQcFfEEQ1ePBxbf6wvzkPMEqKYHhPYv8boI1Rc/24d05d0dbc8eeaff656f0cfc9befa45d_4IMOqpOwCr.icns

https://macosicons.com/#/?icon=4IMOqpOwCr
(Windows 11 File Explorer)
Thanks in advance
